dask/optimize.py

```diff
@@ -45,7 +47,13 @@ def fuse(dsk):
     >>> d = {'a': 1, 'b': (inc, 'a'), 'c': (inc, 'b')}
     >>> fuse(d)  # doctest: +SKIP
     {'c': (inc, (inc, 1))}
+    >>> fuse(d, keys=['b'])  # doctest: +SKIP
+    {'b': (inc, 1), 'c': (inc, 'b')}
     """
+    if isinstance(keys, (set, list)):
+        keys = set(keys)
+    elif keys is not None:
+        keys = set([keys])
```

Comment:
This fails for tuples of keys, which is something that I could easily see myself passing in and then wondering why everything failed :/